### DISCUSSION TITLE
Mouse wheel + modifier to control knobs [SOLVED]

### DISCUSSION THREAD

**Discussion by alltiagocom** [DISTRHO/Cardinal#514] [Ideas]
It would be great if we could use the mouse wheel with a modifier key (maybe OPTION/ALT on Mac) to increase/decrease the value on a knob, besides clicking and dragging up/down.

**Comment by cmorse01** [comment]
Go into the view menu and turn on scroll wheel knob control.  There on Linux, presume Mac is same.

**Reply by alltiagocom** [reply]
Thank you so much. For some reason I saw that as a title for what's underneath (the slider to control the sensitivity). I didn't see it as an option that could be clicked to activate the knob control. Really appreciate it!
